Blockstack/Classes/Blockstack.swift

```diff
@@ -9,23 +9,27 @@ import Foundation
 
 open class Blockstack {
     
-    private var instance: BlockstackInstance?
+    private var authenticationModule: BlockstackAuthentication
+    private var storageModule: BlockstackStorage
```

Comment:
`Blockstack` contains references to all the modules and serves as a façade for the lib user.

It'll also help in clearly designing all the available methods to the user without the danger of leaking some methods.